src/_pytest/compat.py

```diff
@@ -342,6 +342,12 @@ def final(f):
         return f
 
 
+# TODO: sealed classes are not supported yet
```

Comment:
is it worth trying to pre-empt `@sealed` existing? or just move the TODO: into warning_types.py?

Reply:
As @bluetech commented, it is fine to just go ahead and remove it. 👍